tests/test_noninteractive_mode.py

```diff
@@ -63,6 +63,47 @@ def get_file_contents(file_path):
             raise e
 
 
+class TestNonInteractiveDbModification:
+    """
+    Test DB modification in non-interactive mode.
+    """
+
+    @staticmethod
+    @pytest.fixture(scope="class")
+    def test_db():
+        """ pytest fixture that creates test db """
+        test_db = create_test_db()
+        yield test_db
+        clean_up_test_db(test_db)
+
+    @staticmethod
+    @pytest.fixture(scope="function")
+    def mssqlcli(test_db):
+        return create_mssql_cli(interactive_mode=False, database=test_db)
+
+    """
+    TODO: TESTS TO CONSIDER:
```

Comment:
All of these scenarios would take the same code path, no? What value does each scenario bring?

Reply:
Good point. It's the same code path, so perhaps no added value, at least for testing non-interactive mode. Any you can think of that may add value?